web/frontend/src/components/slam.vue

```diff
@@ -84,7 +81,7 @@ const fetchSLAMPose = (name: string): Promise<commonApi.Pose> => {
     props.client.slamService.getPosition(
       req,
       new grpc.Metadata(),
-      (error: ServiceError, res: slamApi.GetPositionResponse): void => {
+      (error: ServiceError | null, res: slamApi.GetPositionResponse | null): void => {
```

Comment:
is it fair to say that one & only one of `error` and `res` will be defined?

Reply:
this is unclear

in the logic below it looks like we *are* handling this case a "successful" response is null. (i.e. we are calling `resp!` instead of just `resp`).